scripts/add_nerc_rates.py

```diff
@@ -0,0 +1,52 @@
+import os
+from datetime import datetime
+from decimal import Decimal
+
+from jinja2 import Environment, FileSystemLoader
+
+from nerc_rates import load_from_url, load_from_file
+
+SU_TYPE_LIST = [
+    "CPU",
+    "GPUA100",
+    "GPUA100SXM4",
+    "GPUV100",
+    "GPUK80",
+    "GPUH100",
+    "BM FC430",
+    "BM FC830",
+    "BM GPUA100SXM4",
+    "BM GPUH100"
+]
+
+SU_RESOURCETYPE_LIST = [
+    "vCPUs",
+    "vGPUs",
+    "RAM"]
+
+TEMPLATE_FILE_LIST = [
+    "docs/get-started/cost-billing/how-pricing-works.md",
+    "docs/get-started/cost-billing/pricing-for-bare-metal-machines.md"
+]
```

Comment:
I'm not sure how I feel about hardcoding the list of files into this script. For an alternative take on this, take a look at <https://github.com/nerc-project/nerc-docs/compare/main...larsks:nerc-docs:fetch-nerc-rates?expand=1>, in which I modify the script to take command line arguments and add a `Makefile` to drive the templating.

I guess if it's only ever going to be these two files, maybe it doesn't really matter.

Reply:
Do you want me to incorporate your sample commit into my commit? Or should I submit this PR as two commits, one by me and one by you?

Reply:
I am not suggesting that you need to incorporate my changes. I am offering it as an alternative example. If you think you like it better than this one, feel free to incorporate it. If you don't think it adds value, feel free to ignore it.

Reply:
@larsks For now, I will leave the list of files in the script. If in the future, it turns out we have a lot more `md` files we want to add templating to, I'll make sure to remember your suggestion.